### PR TITLE
Add the reporter blogpost for CVE-2022-27909

### PR DESCRIPTION
**Pull request by zero-24**

### CODE REVIEW

File: 2022/27xxx/CVE-2022-27909.json [hunk 1]
```diff
@@ -59,6 +59,11 @@
                 "url": "https://www.jdownloads.com/index.php/downloads/download/57-jdownloads-3-9.html",
```
Comment (cve-team) [inline 2022/27xxx/CVE-2022-27909.json:59]: 'https://hackerhood.redhotcyber.com/cve-2022-27909-jdownloads/ ' does not match '^(ftp|http)s?://\\S+$'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/27xxx/CVE-2022-27909.json (hunk 1 above)
Comment (zero-24) [reply 2022/27xxx/CVE-2022-27909.json:59]: @cve-team can you please double check that regex? The URL looks valid to me?

Comment (anthnysingleton) [reply 2022/27xxx/CVE-2022-27909.json:59]: The issue is with the trailing white space at the end of the string. Can you remove it with an update?

Comment (zero-24) [reply 2022/27xxx/CVE-2022-27909.json:59]: I'm sorry it has been fixed

**Pull request merged**